databricks/koalas/series.py

```diff
@@ -3161,6 +3161,146 @@ def idxmin(self, skipna=True):
+                weight    250.0
+                length      1.5
+        falcon  speed     320.0
+                weight      1.0
+                length      0.3
+        Name: 0, dtype: float64
+
+        >>> s.pop('lama')
+        speed      45.0
+        weight    200.0
+        length      1.2
+        Name: 0, dtype: float64
+
+        >>> s
+        cow     speed      30.0
+                weight    250.0
+                length      1.5
+        falcon  speed     320.0
+                weight      1.0
+                length      0.3
+        Name: 0, dtype: float64
+
+        Also support for MultiIndex with several indexs.
+
+        >>> midx = pd.MultiIndex([['a', 'b', 'c'],
+        ...                       ['lama', 'cow', 'falcon'],
+        ...                       ['speed', 'weight', 'length']],
+        ...                      [[0, 0, 0, 0, 0, 0, 1, 1, 1],
+        ...                       [0, 0, 0, 1, 1, 1, 2, 2, 2],
+        ...                       [0, 1, 2, 0, 1, 2, 0, 1, 2]]
+        ...  )
+        >>> s = ks.Series([45, 200, 1.2, 30, 250, 1.5, 320, 1, 0.3],
+        ...              index=midx)
+        >>> s
+        a  lama    speed      45.0
+                   weight    200.0
+                   length      1.2
+           cow     speed      30.0
+                   weight    250.0
+                   length      1.5
+        b  falcon  speed     320.0
+                   weight      1.0
+                   length      0.3
+        Name: 0, dtype: float64
+
+        >>> s.pop(('a', 'lama'))
+        speed      45.0
+        weight    200.0
+        length      1.2
+        Name: 0, dtype: float64
+
+        >>> s
+        a  cow     speed      30.0
+                   weight    250.0
+                   length      1.5
+        b  falcon  speed     320.0
+                   weight      1.0
+                   length      0.3
+        Name: 0, dtype: float64
+        """
+        from databricks.koalas.indexes import MultiIndex
+        if not isinstance(item, (str, tuple)):
+            raise ValueError("'key' should be string or tuple that contains strings")
+        if isinstance(item, str):
+            item = (item,)
+        if not all(isinstance(index, str) for index in item):
+            raise ValueError("'key' should have index names as only strings "
+                             "or a tuple that contain index names as only strings")
+
+        if isinstance(self.index, MultiIndex):
+            cols = self._internal.index_scols[len(item):] + [self._internal.scol_for(self.name)]
+            rows = [self._internal.scols[level] == index
+                    for level, index in enumerate(item)]
+            sdf = self._internal.sdf \
+                .select(cols) \
+                .where(reduce(lambda x, y: x & y, rows))
+
+            # if sdf has only a data column, return data only without frame
+            if len(self._index_map) == len(item):
+                if sdf.count() != 0:
```

Comment:
Actually I don't like to address 3) since it invokes `count()`.

Reply:
Btw, why does this happen only MultiIndex case?

Reply:
i think it was my mistake. they don't need to have different behavior between single / multi index.
fixed them!

Reply:
and thanks for reviewing !!

Reply:
ah, and i fixed them to don't use count() also, but if you wanna make it works exactly as same as pandas (raises KeyError rather than return empty series), please fell free to tell me anytime.

Reply:
I'd prefer the previous behavior: Koalas returns empty Series